src/test/fuzz/util/net.cpp

```diff
@@ -182,54 +182,54 @@ ssize_t FuzzedSock::Recv(void* buf, size_t len, int flags) const
         EWOULDBLOCK,
     };
     assert(buf != nullptr || len == 0);
+
+    // Do the latency before any of the "return" statements.
+    if (m_fuzzed_data_provider.ConsumeBool() && std::getenv("FUZZED_SOCKET_FAKE_LATENCY") != nullptr) {
+        std::this_thread::sleep_for(std::chrono::milliseconds{2});
```

Comment:
Unrelated to this PR but manual timeouts in a fuzz test are a bad idea. If a target requires the use of `GetTime` (or similar) then the harness needs to set mock time (as the test is otherwise non-deterministic).

We should get rid of this and replace it with calls to `SetMockTime` (we might need to make mock time more granular, i.e. currently it only supports seconds iirc).